Gruntfile.js

```diff
@@ -58,8 +59,31 @@ module.exports = function(grunt)
           }
         ]
       }
+    },
+    prettier: {
+      options: {
+        progress: true,
+        singleQuote: true
+      },
+      tests: {
+        src: [
+          'test/**/*.js'
+        ]
+      },
+      // src: {
```

Comment:
These here for any reason?

Reply:
Yeah I wanted it there, but not usable yet. I want to get all of the testing stuff out of the way and then issue a big PR where we reformat everything with `prettier`. I just didn't want it to get ran automatically before we do that.